prac_08/convert_miles_km.py

```diff
@@ -0,0 +1,39 @@
+from kivy.app import App
+from kivy.lang import Builder
+from kivy.properties import StringProperty
+
+MILES_TO_KILOMETRES = 1.60934
+EMPTY_DATA = 0
+
+
+class ConvertMilesApp(App):
+    """App for converting miles to kilometres."""
+    output_text = StringProperty()
+
+    def build(self):
+        """Use Kivy file to create Kivy app."""
+        self.title = "Convert miles to kilometres"
+        self.root = Builder.load_file('convert_miles_km.kv')
```

Comment:
I think you need to add "self.output_text = '32.1868'" to show that you have used StringProperty().

Reply:
Noted !
Thanks you